bin/erlnodectl

```diff
@@ -97,43 +100,42 @@ start () \
   su >/dev/null $csetsep -l $dashs "$user" -c                         \
     "env cookie=\"$cookie\" erlstart-run-erlang \"$ERLSTART_CONFIG_FILE\""
 
+  ERL_CRASH_DUMP=/dev/null; export ERL_CRASH_DUMP
+  ERL_CRASH_DUMP_SECONDS=0; export ERL_CRASH_DUMP_SECONDS
```

Comment:
Sometimes the crash dumps are useful and you want to wait for them, maybe we should allow these to be overridden as well?